content/en/get-started/deploy-your-app.md

```diff
@@ -10,50 +10,61 @@ tags: ["Deployment", "Quickstart"]
 
 ## Learning objectives
 
-In this guide, you build on what you learned in the  CD-as-a-Service [Quickstart]({{< ref "get-started/quickstart" >}}) by deploying your own app using the deployment configuration from the Quickstart.
+1. [Create a deployment config file](#create-a-deployment-config-file)
+2. [Deploy your app](#deploy-your-app)
+3. [Monitor your deployment](#monitor-your-deployment)
 
-1. [Create your deployment config file](#create-a-deployment-config-file)
-1. [Deploy your app](#deploy-your-app)
-
-
->If you prefer a web-based, interactive tutorial, see the CD-as-a-Service Console's [**Deploy Your Own App** tutorial](https://next.console.cloud.armory.io/getting-started).
 
 ## {{% heading "prereq" %}}
 
```

Comment:
I think we should include a point:

```
* You are familiar with the material in the [Quickstart]({{< ref "get-started/quickstart" >}}.
```

If we don't, then we should not assume the user knows deployment file syntax, how to issue manual approvals, or where to find the deploy preview link.